tests/by-util/test_env.rs

```diff
@@ -179,27 +190,33 @@ fn test_change_directory() {
     assert_eq!(out.trim(), temporary_path.as_os_str())
 }
 
-// no way to consistently get "current working directory", `cd` doesn't work @ CI
-// instead, we test that the unique temporary directory appears somewhere in the printed variables
 #[cfg(windows)]
 #[test]
 fn test_change_directory() {
     let scene = TestScenario::new(util_name!());
     let temporary_directory = tempdir().unwrap();
+
     let temporary_path = temporary_directory.path();
+    let temporary_path = temporary_path
+        .strip_prefix(r"\\?\")
+        .unwrap_or(temporary_path);
 
-    assert_ne!(env::current_dir().unwrap(), temporary_path);
+    let env_cd = env::current_dir().unwrap();
+    let env_cd = env_cd.strip_prefix(r"\\?\").unwrap_or(temporary_path);
```

Comment:
I assume you meant `unwrap_or(env_cd)`? The assertion error is because you assert the paths are not equal, but they are.

Reply:
I thought the assertion error was in a different line... Assumed the problem was here: `assert_eq!(out.trim(), temporary_path.as_os_str())` and didn't stop to look there... Thanks for pointing it out